share/wake/lib/prim/prim.wake

```diff
@@ -0,0 +1,401 @@
+package prim_wake
+
+export def prim_abs: Integer => Integer =
```

Comment:
I hate to be that guy, but we should probably write comments for these

Reply:
This file is generated from the script and isn't supposed to directly used. Instead they are supposed to be wrapped by other properly commented stdlib functions. 

I don't think we could generate the comments for these but I suppose we could consider the script a "run once" script then manually update it moving forward. 

Regardless I don't personally have the context for everything here and would need help if we decided that this should be commented

Reply:
> supposed to be wrapped by other properly commented stdlib functions.

I'm not sure I understand the use-case then, I had assumed this was to replace ad-hoc usage of `prim` in regular user code, as there are some times the need to use prim for cases that the stdlib doesnt handle